<ui version="4.0" >
 <author>werner schweer</author>
 <comment>midi sync
configuration dialog</comment>
 <exportmacro></exportmacro>
 <class>MidiSyncConfigBase</class>
 <widget class="QDialog" name="MidiSyncConfigBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>4</hsizetype>
    <vsizetype>4</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>MusE: Midi Sync</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="syncMaster" >
     <property name="title" >
      <string>Master</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="mtcSync" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Midi Time Code (MTC)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mcSync" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Midi Clock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="midiMachineControl" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Midi Machine Control (MMC)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel7" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="dstDevId" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>device id</string>
          </property>
          <property name="maximum" >
           <number>127</number>
          </property>
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="value" >
           <number>127</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="syncSlave" >
     <property name="title" >
      <string>Slave</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="acceptMTCCheckbox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>accept MTC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="acceptMCCheckbox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>accept Midi Clock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="acceptMMCCheckbox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>accept MMC</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel7_2" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="srcDevId" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>device id</string>
          </property>
          <property name="maximum" >
           <number>127</number>
          </property>
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="value" >
           <number>127</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="GroupBox13" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>MTC</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mtcSyncType" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text" >
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>30D</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>30N</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>407</width>
            <height>69</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mtcOffH" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>hour</string>
          </property>
          <property name="maximum" >
           <number>23</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel2" >
          <property name="text" >
           <string>h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mtcOffM" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>minute</string>
          </property>
          <property name="maximum" >
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel3" >
          <property name="text" >
           <string>m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mtcOffS" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>second</string>
          </property>
          <property name="maximum" >
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel4" >
          <property name="text" >
           <string>s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mtcOffF" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>frame</string>
          </property>
          <property name="maximum" >
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel5" >
          <property name="text" >
           <string>f</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mtcOffSf" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>subframe</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>acceptMTCCheckbox</tabstop>
  <tabstop>acceptMCCheckbox</tabstop>
  <tabstop>mtcSync</tabstop>
  <tabstop>mcSync</tabstop>
  <tabstop>midiMachineControl</tabstop>
  <tabstop>dstDevId</tabstop>
  <tabstop>mtcOffH</tabstop>
  <tabstop>mtcOffM</tabstop>
  <tabstop>mtcOffS</tabstop>
  <tabstop>mtcOffF</tabstop>
  <tabstop>mtcOffSf</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>syncSlave</sender>
   <signal>toggled(bool)</signal>
   <receiver>acceptMTCCheckbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>377</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel" >
     <x>341</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncSlave</sender>
   <signal>toggled(bool)</signal>
   <receiver>acceptMCCheckbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>365</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel" >
     <x>361</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncSlave</sender>
   <signal>toggled(bool)</signal>
   <receiver>acceptMMCCheckbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>392</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel" >
     <x>328</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncSlave</sender>
   <signal>toggled(bool)</signal>
   <receiver>srcDevId</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>366</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel" >
     <x>375</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mcSync</sender>
   <signal>toggled(bool)</signal>
   <receiver>mcSync</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>255</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel" >
     <x>255</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mtcSync</sender>
   <signal>toggled(bool)</signal>
   <receiver>mtcSync</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>255</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel" >
     <x>255</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncMaster</sender>
   <signal>toggled(bool)</signal>
   <receiver>midiMachineControl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel" >
     <x>255</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncMaster</sender>
   <signal>toggled(bool)</signal>
   <receiver>dstDevId</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel" >
     <x>175</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncMaster</sender>
   <signal>toggled(bool)</signal>
   <receiver>mtcSync</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel" >
     <x>255</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncMaster</sender>
   <signal>toggled(bool)</signal>
   <receiver>GroupBox13</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>53</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel" >
     <x>41</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
